g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/conf/CollectSourceValidationInterceptor.java

```diff
@@ -31,11 +31,13 @@
 	
 	private static Logger LOGGER = LoggerFactory.getLogger(CollectSourceValidationInterceptor.class);
 	
-	public CollectSourceValidationInterceptor(List<String> sourceLocales, Map<String, List<String>> allowListMap) {
+	public CollectSourceValidationInterceptor(List<String> sourceLocales, Map<String, List<String>> allowListMap, String sourceLocalesStr) {
 		this.sourceLocales = sourceLocales;
 		this.allowList = allowListMap;
+		this.sourceLocalesStr = sourceLocalesStr;
```

Comment:
Need to know the purposes of sourceLocales and sourceLocaleStr.

Reply:
sourceLocales use validate the collect resource  locale， sourceLocaleStr use response String when validate false